runtime/mangata-kusama/src/lib.rs

```diff
@@ -160,11 +160,11 @@ impl_opaque_keys! {
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("mangata-parachain"),
 	impl_name: create_runtime_str!("mangata-parachain"),
-	authoring_version: 10,
-	spec_version: 10,
+	authoring_version: 11,
```

Comment:
This should be incremented only when preparing the RC, so we don't have conflicts.

Reply:
We need to do this, so that it doesn't fail ci.

Reply:
Understand. Didn't know. Noting it down for some future CI improvements.